cvat-ui/src/components/annotation-page/canvas/canvas-wrapper3D.tsx

```diff
@@ -35,6 +42,13 @@ const CanvasWrapperComponent = (props: Props): ReactElement => {
     const sideView = useRef<HTMLCanvasElement>(null);
     const frontView = useRef<HTMLCanvasElement>(null);
```

Comment:
These refs are applied to HTMLDivElement, not HTMLCanvasElement. 
Probably need to fix these generic types to useRef<HTMLDivElement>